src/store.ts

```diff
@@ -311,4 +337,22 @@ export class Store implements Cache {
   ): void {
     writeFragment(this, dataFragment, data, variables);
   }
+
+  hydrateData(data: object, storage: StorageAdapter) {
+    for (const key in data) {
+      switch (key.charCodeAt(0)) {
+        case 99:
+          this.writeConnection(key.slice(2), data[key][1], data[key][0]);
```

Comment:
There’s a small bug on this line. `data[key]` here will be an array and not a single connection